easybuild/easyconfigs/q/QUAST/QUAST-5.2.0-foss-2021a.eb

```
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'QUAST'
+version = '5.2.0'
+
+homepage = 'https://github.com/ablab/quast'
+description = """QUAST evaluates genome assemblies by computing various metrics.
+It works both with and without reference genomes. The tool accepts multiple
+assemblies, thus is suitable for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Perl', '5.32.1'),
+    ('matplotlib', '3.4.2'),
+    ('Java', '11.0.18', '', SYSTEM),
```

```suggestion
    ('Java', '11', '', SYSTEM),
```
the version wrapper should always be used for Java

---

Thanks @Micket for swiftly taking this PR over. The commit 609f4cf28644e4a31f96b8c3ce04411540c2b89c shall address your comment.